### PR TITLE
Bug 1452606 - Do not allow non-Admin groups to change Admin and Peers…

### PR DESCRIPTION
**Pull request by MichaelKohler**
… groups

### CODE REVIEW

**Comment by MichaelKohler** [comment]
@akatsoulas this is waiting for your final review :) Thanks!

File: remo/profiles/views.py [hunk 1]
```diff
@@ -87,8 +87,14 @@ def profile_date_form_validation(form):
                       'Onboarding': 'onboarding_group',
                       'Newsletter': 'newsletter_group'}
 
+            admin_only_groups = ['Admin', 'Peers']
+
+            user_is_admin = request.user.groups.filter(name='Admin').exists()
             for group_db, group_html in groups.items():
                 if Group.objects.filter(name=group_db).exists():
+                    if group_db in admin_only_groups and not user_is_admin:
```
Comment (akatsoulas) [inline remo/profiles/views.py]: I would suggest to completely hide the groups from users that shouldn't be able to edit them.

[the remark this answers is not in the document]
Comment (MichaelKohler) [reply remo/profiles/views.py]: @akatsoulas this is updated and ready for your review again :)

**Comment by akatsoulas** [comment]
Nice work! Please squash before merging.

**Pull request merged**